am-lib/src/main/java/uk/gov/hmcts/reform/amlib/models/RemoveExplicitAccessRecord.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.reform.amlib.models;
+
+import lombok.Builder;
+import lombok.Data;
+import org.jdbi.v3.core.mapper.reflect.JdbiConstructor;
+
+@Data
+@Builder
+public class RemoveExplicitAccessRecord {
```

Comment:
Name of the class sounds a bit like verb which is wrong.

I suggest renaming it to `ExplicitAccessMetadata` and creating subclass `ExplicitAccessRecord` out of it.


```
ExplicitAccessRecord extends ExplicitAccessMetadata {
  // permissions only
}
```

This way we can avoid duplicating bit of code.